src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -37,6 +38,9 @@ public Manufacturer create(Manufacturer manufacturer) {
 
     @Override
     public Optional<Manufacturer> get(Long id) {
+        if (id == null) {
+            throw new DataProcessingException("The id cannot be null " + id, new SQLException());
+        }
```

Comment:
Actually, @nacenik meant a different thing. He wrote you about using orElseGet() method with an informative error message instead of simple get() within service class method. So, please remove this null check and accomplish what I described